src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -101,80 +86,129 @@ const props = defineProps({
     default: '',
   },
   /**
-   * Use this props to set item of item.
+   * Use this props to set id of item.
    */
   id: {
     type: String,
     default: '',
   },
+  /**
+   * Use this props to pass attrs for label UiText.
+   */
+  textLabelAttrs: {
+    type: Object as PropType<MultipleAnswerLabelAttrs>,
+    default: () => ({}),
+  },
   /**
    * Use this props to pass attrs for info UiButton.
    */
   buttonInfoAttrs: {
```

Comment:
@ArkadiuszKrzymienski after go to review UiMultipleChoices I think we should keep the same API for both elements and
* keep buttonInfoAttrs etc...
* translate buttonInfoAttrs etc... to suffixAttrs
When we use name Button Info in UiMultipleChoices we should use the same name here (the same feature)